docker-compose.yaml

```diff
@@ -89,6 +89,15 @@ services:
     depends_on:
       - db
 
+  dataset_file_metadata:
```

Comment:
`datasets_worker` may be better since we are going to do more things than just metadata extraction

Reply:
Yes, it is a better name